packages/components/psammead-media-player/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version       | Description                                                                                                                  |
 | ------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| 2.0.1 | [PR#2424](https://github.com/bbc/psammead/pull/2424) Add srcset to placeholder image
```

Comment:
This PR changes the way placeholder images are loaded, I think this should be considered a `minor` update rather than a `patch`.